packages/local_auth/local_auth_darwin/darwin/Classes/FLALocalAuthPlugin.m

```diff
@@ -153,6 +174,27 @@ - (void)showAlertWithMessage:(NSString *)message
           dismissButtonTitle:(NSString *)dismissButtonTitle
      openSettingsButtonTitle:(NSString *)openSettingsButtonTitle
                   completion:(FLADAuthCompletion)completion {
+#if TARGET_OS_OSX
+  NSAlert *alert = [[NSAlert alloc] init];
+  [alert setMessageText:message];
+  [alert addButtonWithTitle:dismissButtonTitle];
+  if (openSettingsButtonTitle != nil) {
+    [alert addButtonWithTitle:openSettingsButtonTitle];
+  }
+  NSWindow *keyWindow = self.registrar.view.window;
```

Comment:
"Key window" is a specific term with a defined meaning in macOS development; this is not the key window. It's the window containing the Flutter content.

There's no ambiguity here though, so the variable can just be called `window`.

Reply:
Fixed